application-xpoll-ui/src/main/resources/XPollCode/RecentPollsPanel.xml

```diff
@@ -20,15 +20,15 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.2" reference="XPollCode.RecentPollsPanel" locale="">
-  <web>XPollCode</web>
+<xwikidoc version="1.2" reference="XPoll.XPollCode.RecentPollsPanel" locale="">
```

Comment:
Same as above. The page should be in the ``Code`` subspace. But what's even more important is that the page reference needs to match the location of the page on the file system. The page is still in ``resources/XPollCode/``. It should be in ``/resources/XPoll/Code/``.

Reply:
So Code should be moved from resouces/ to resouces/XPoll space?

Reply:
You just need to keep the path you have in the exported XAR. The issue is that you copied the XML from the XAR where is has the new path to the source tree where is has the old path.